Test/c++/arrays.dfy

```diff
@@ -0,0 +1,96 @@
+newtype{:nativeType "uint"} uint32 = i:int | 0 <= i < 0x100000000
```

Comment:
The Dafny implementation has hooks for assigning native types automatically. If they are used, you shouldn't need the explicit `:nativeType` annotation here (or in the other test files).

Reply:
Ah, interesting, I hadn't realized that.  I'll experiment with removing the annotations.